pkg/lockfile/parse-apk-installed.go

```diff
@@ -0,0 +1,99 @@
+package lockfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const AlpineEcosystem Ecosystem = "Alpine"
+
+func groupApkPackageLines(scanner *bufio.Scanner) [][]string {
+	var groups [][]string
+	var group []string
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if line == "" {
+			if len(group) > 0 {
+				groups = append(groups, group)
+			}
+			group = make([]string, 0)
+			continue
+		}
+		group = append(group, line)
+	}
+
+	if len(group) > 0 {
+		groups = append(groups, group)
+	}
+
+	return groups
+}
+
+func parseApkPackageGroup(group []string, pathToLockfile string) PackageDetails {
+	var pkg = PackageDetails{}
+
+	pkg.Ecosystem = AlpineEcosystem
+	pkg.CompareAs = AlpineEcosystem
```

Comment:
nit: lets set these in the assignment

```suggestion
	var pkg = PackageDetails{
		Ecosystem: AlpineEcosystem,
		CompareAs: AlpineEcosystem,
	}
```